app/auth/signIn/SignInProviders.tsx

```diff
@@ -0,0 +1,40 @@
+'use client';
+
+import { Button } from '@/components/ui/button';
+import { signIn } from 'next-auth/react';
+
+type Provider = {
+    id: string;
+    name: string;
+};
+
+export default function SignInProviders({
+    providers,
+}: {
+    providers: Provider[] | null;
+}) {
+    if (!providers || providers.length === 0)
+        return (
+            <p className="text-destructive">
+                No authentication providers available.
+            </p>
+        );
+
+    return (
+        <>
+            {providers.map((provider) => (
+                <Button
+                    key={provider.id}
+                    className="rounded-full bg-blue-500 px-20 py-6 font-semibold text-white hover:bg-blue-600"
```

Comment:
```suggestion
                    className="px-20 py-6"
                    variant={'secondary'}
```
Since we do not have provider-logo and colors, we should consider just keeping the buttons as the secondary variant for UI consistency.

We should probably discuss whether we should design the buttons with provider-logo and colors or not.

Reply:
I agree. I think anyways that using the secondary variant is better than what it is already. We can figure if we want logos or not later/in its own issue/pr.